.gitignore

```diff
@@ -1,3 +1,5 @@
 *.pyc
 __pycache__
-venv
\ No newline at end of file
+venv
+lmnopEnv
+*.sqlite
```

Comment:
I've added env and .vscode as suggested by Clara. Before we merge your code in we need that here too.